pdf_barcodes/README.md

```diff
@@ -0,0 +1 @@
+This is the folder that will contain the PDF generated. To be moved to a Object Storage platform in the future.
```

Comment:
I think it's more convenient to create GH issues instead. They can be tracked and don't get lost.